e2e_tests/tests/experiment/test_tasks_raw_csv.py

```diff
@@ -0,0 +1,97 @@
+import re
+from datetime import datetime, timezone
+
+import pytest
+import requests
+
+from determined.common import api
+from determined.common.api.bindings import experimentv1State
+from tests import cluster as clu
+from tests import command as cmd
+from tests import config as conf
+from tests import experiment as exp
+
+
+# Create a No_Op experiment and Check training/validation times
+@pytest.mark.e2e_cpu
+def test_experiment_capture() -> None:
+    start_time = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
+
+    experiment_id = exp.create_experiment(
+        conf.fixtures_path("no_op/single.yaml"), conf.fixtures_path("no_op")
+    )
+    exp.wait_for_experiment_state(experiment_id, experimentv1State.STATE_COMPLETED)
+
+    end_time = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
+    r = api.get(
+        conf.make_master_url(),
+        f"/resources/allocation/tasks-raw?timestamp_after={start_time}&timestamp_before={end_time}",
+    )
+    assert r.status_code == requests.codes.ok, r.text
+
+    # Check if a trial entry exists for experiment that just ran
+    matches = re.search(f".+,TRIAL,.+,{experiment_id},.+", r.text)
+    assert matches is not None
+
+    # Confirm valid training and validation time were captured for the given experiment
+    training_time, validation_time = matches.group(0).split(",")[-3:-1]
```

Comment:
maybe parse it in a more stable way, as a CSV, so potential column changes don't affect it?

```suggestion
    reader = csv.DictReader(StringIO(r.text))
    matches = [row for row in reader if int(row["experiment_id"]) == experiment_id]
    assert len(matches) == 1
    training_time, validation_time = matches[0]["training_time"], matches[0]["validation_time"]
```

Reply:
Makes a lot of sense! Committing that change